src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -32,6 +33,17 @@ function forceUpdateIfMounted() {
   }
 }
 
+function warnIfValueIsNull(props) {
+
+  if (props != null && props.value === null && !didWarnValueNull) {
+    warning(false, '`value` prop on `input` should not be null.' +
```

Comment:
This formatting doesn't match our style, see any other warning or invariant callsite for examples.

Also, please remove the empty line above the condition.